explorer.go

```diff
@@ -0,0 +1,257 @@
+package main
+
+import (
+	"html/template"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/decred/dcrutil"
+
+	apitypes "github.com/dcrdata/dcrdata/dcrdataapi"
+	"github.com/decred/dcrd/dcrjson"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+	"github.com/rs/cors"
+)
+
+type explorerMux struct {
+	*chi.Mux
+}
+
+func (c *appContext) explorerUI(w http.ResponseWriter, r *http.Request) {
+	helpers := template.FuncMap{
+		"getTime": getTime,
+	}
+	explorerTemplate, _ := template.New("explorer").Funcs(helpers).ParseFiles("views/explorer.tmpl", "views/extras.tmpl")
```

Comment:
There was a suggestion from gozart to make this template parsing configurable with an environment variable. I can see how designers wouldnt want to have to restart the daemon while doing edits.

Reply:
Agreed that it's absolutely necessary to avoid restarting just to update templates.  The ability to signal a reload if the templates with SIGUSR1 was added for this purpose: https://github.com/dcrdata/dcrdata/pull/37/commits/2460e455fda8f9cac8905f217af7151432fbe04b
@gozart1 let me knew if this addresses the need

Reply:
Added this to the explorer as well